prowler/providers/microsoft365/services/entra/entra_admin_users_sign_in_frequency_enabled/entra_admin_users_sign_in_frequency_enabled.py

```diff
@@ -0,0 +1,72 @@
+from prowler.lib.check.models import Check, CheckReportMicrosoft365
+from prowler.providers.microsoft365.services.entra.entra_client import entra_client
+from prowler.providers.microsoft365.services.entra.entra_service import (
+    AdminRoles,
+    ConditionalAccessPolicyState,
+)
+
+
+class entra_admin_users_sign_in_frequency_enabled(Check):
+    """Check if Conditional Access policies enforce sign-in frequency for admin users.
+
+    This check ensures that administrators have a sign-in frequency policy enabled
+    and that persistent browser session settings are correctly configured.
+    """
+
+    def execute(self) -> list[CheckReportMicrosoft365]:
+        """Execute the check to validate sign-in frequency enforcement for admin users.
+
+        Returns:
+            list[CheckReportMicrosoft365]: A list containing the results of the check.
+        """
+        findings = []
+
+        report = CheckReportMicrosoft365(
+            metadata=self.metadata(),
+            resource=entra_client.conditional_access_policies,
```

Comment:
```suggestion
            resource={},
```
We cannot pass a dict as a resource.